WordPress/src/main/java/org/wordpress/android/ui/deeplinks/DeepLinkNavigator.kt

```diff
@@ -114,5 +117,7 @@ class DeepLinkNavigator
         object OpenLoginPrologue : NavigateAction()
         data class OpenJetpackForDeepLink(val action: String?, val uri: UriWrapper) : NavigateAction()
         object OpenJetpackStaticPosterView : NavigateAction()
+        data class OpenMediaForSite(val site: SiteModel) : NavigateAction()
+        object OpenMedia : NavigateAction()
```

Comment:
Updated the logic to use `ActivityNavigator` here. Commit -  61fc7694f3413e5eb92524256c5b726c07698794